source/v4/teams.yaml

```diff
@@ -271,3 +271,37 @@
           $ref: '#/responses/Forbidden'
         '404':
           $ref: '#/responses/NotFound'
+
+  /users/{user_id}/teams/unread
+    get:
+      tags:
+        - teams
+      summary: Get the count for unread messages and mentions in the teams the user is a member of
```

Comment:
Can we shorten this to `Get team unreads for a user`? It's better to keep the summaries short since they'll appear in a sidebar